### ISSUE TITLE
DebugMenu improvements

### ISSUE THREAD

**Issue by lstebner** [jeremyckahn/farmhand#100]
the current debug menu offers an instant $10k, and a "buy 10 carrots" option, but it would be helpful to be able to do more stuff here like buy any crop, and maybe any amount of them.

**Comment by jeremyckahn** [comment]
Yeah, I agree that the current debug menu is pretty useless at this point. I used it a lot more earlier on in Farmhand's development. I don't really use it all anymore and tend to lean on React Dev Tools for more development/debugging. I'd be fine with either adding to this feature or removing it entirely.

**Comment by lstebner** [comment]
I actually like the debugmenu a lot. i like that it goes through the real interactions and triggers the side effects like checking achievements/experience. but if it seems like a pain to maintain i defer to your opinion. maybe you could just document how you do testing for things like gaining levels, and getting recipes unlocks if we get rid of this

**Comment by jeremyckahn** [comment]
It's not hard to maintain at this point, I just don't really use it personally. I'm good with keeping and extending it. I'm happy to help out with PRs for improving it!

**Comment by lstebner** [comment]
i forgot the debug menu even existed. i typically just modify state to test and i think @jeremyckahn recommends this approach. i'm going to close this issue

**Comment by jeremyckahn** [comment]
@lstebner Good idea. I think the Debug menu has long outlived its usefulness and is worth removing as a cleanup task.
